tests/test_git_info.py

```diff
@@ -0,0 +1,23 @@
+from pathlib import Path
+from data_pipeline_api.git_info import (
+    get_repo,
+    get_repo_info,
+    get_path_relative_to_repo,
+    Repo,
+    RepoInfo,
+    git_installed
+)
+
+
+# We can only test this module if git is installed.
+if git_installed:
```

Comment:
I'm actually going to change this again, to check that the interface does the expected thing when git is not installed.

Reply:
done